chaos-monkey-dependencies/pom.xml

```diff
@@ -95,12 +95,30 @@
                 <version>${lombok.version}</version>
                 <scope>provided</scope>
             </dependency>
+            <dependency>
+                <groupId>org.junit.jupiter</groupId>
+                <artifactId>junit-jupiter-api</artifactId>
+                <version>${junit.jupiter.version}</version>
+                <scope>test</scope>
+            </dependency>
             <dependency>
```

Comment:
After all tests are migrated this dependency can be removed.

Reply:
Maybe I'm wrong but don't you mean the junit-vintage-engine?

Reply:
Yes, you're right, I mixed that up. :-)
This can even be cleaned up after moving to Spring Boot 2.2 (https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.2-Release-Notes#junit-5)